CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- Whitelist ips authorized to acess /admin on edxapp, marsha and richie
+
```

Comment:
Would be nice to apply it also to edxec's admin pages and to all pages of mailcatcher and learninglocker?

Reply:
You mean only if there is a haproxy/nginx/whatever in front of openshift ?

Reply:
done for learninglocker but for mailcatcher there is no nginx. I will check if the application allow to do that.